test.disabled/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -2789,11 +2789,13 @@ exit_room_with_status(Config1) ->
     end),
     destroy_room(Config).
 
+%GIVEN Kate in the room, to ensure that the room does not get stopped
+%WHEN Bob sends malformed presence
+%THEN Bob gets kicked. Join Alice to check that Bob is not in the room. Kate and Bob receive presence unavailable.
 kicked_after_sending_malformed_presence(Config1) ->
     AliceSpec = given_fresh_spec(Config1, alice),
     Config = given_fresh_room(Config1, AliceSpec, []),
     escalus:fresh_story(Config, [{bob, 1}, {kate, 1},{alice, 1}], fun(Bob, Kate, Alice) ->
-        %% GIVEN An user is in the room
```

Comment:
Can we keep both a comment before the testcase and these small comments for each operation?